spec/policies/admin_policy_spec.rb

```diff
@@ -15,30 +15,30 @@
 
   describe 'For a member that is a part of a company' do
     let(:members_company) { Company.find_by_company_number('5562728336')}
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(members_company).authorized? }
 
-    it { is_expected.to be_truthy }
+    it { is_expected.to be_falsey }
 
   end
 
   describe 'For a member that is not part of a company' do
-    subject { described_class.new(admin).authorized? }
+    subject { described_class.new(member).authorized? }
```

Comment:
In order to be a `member`, doesn't there have to be a company associated with that member?  Perhaps this test should be for a user who has applied for membership but not yet accepted, using factory ```:user_with_membership_app```?